src/20241004_Multi_ReserveFactorUpdatesMidOctober/ReserveFactorUpdatesMidOctober.md

```diff
@@ -0,0 +1,70 @@
+- Tests: [AaveV2Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV2Ethereum_ReserveFactorUpdatesMidOctober_20241004.t.sol), [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV2Polygon_ReserveFactorUpdatesMidOctober_20241004.t.sol), [AaveV2Avalanche](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV2Avalanche_ReserveFactorUpdatesMidOctober_20241004.t.sol), [AaveV3Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV3Polygon_ReserveFactorUpdatesMidOctober_20241004.t.sol), [AaveV3Optimism](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV3Optimism_ReserveFactorUpdatesMidOctober_20241004.t.sol), [AaveV3Arbitrum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV3Arbitrum_ReserveFactorUpdatesMidOctober_20241004.t.sol), [AaveV3Base](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV3Base_ReserveFactorUpdatesMidOctober_20241004.t.sol), [AaveV3Gnosis](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241004_Multi_ReserveFactorUpdatesMidOctober/AaveV3Gnosis_ReserveFactorUpdatesMidOctober_20241004.t.sol)
+- [Snapshot](https://snapshot.org/#/aave.eth/proposal/0x9cc7906f04f45cebeaa48a05ed281f49da00d89c4dd988a968272fa179f14d06)
+- [Discussion](https://governance.aave.com/t/arfc-increase-bridged-usdc-reserve-factor-across-all-deployments/17787)
```

Comment:
can you add all discussion and snapshot links regarding the changes ( borrow rate and RF on ethereum, avalanche, polygon)

similar to [this one](https://github.com/bgd-labs/aave-proposals-v3/pull/459/files#diff-e3e6ef8f43d8db481e211944baec46178aa364150b6986241202a9c6c54d705eR65-R72)